Yarn install not working due to rn-emoji-keyboard

# Bug Report

**Original Report**: https://gitlab.com/fdroid/fdroiddata/-/merge_requests/10879#note_908604479

Our current fork of rn-emoji-keyboard is using `react-native-svg`: `^9.8.4`, and that is causing issues with `yarn install`, we should upgrade `react-native-svg` to  `^12.1.1`

## Problem 
[comment]: # (A problem description)


#### Expected behavior
`yarn install` should work


#### Actual behavior
`yarn install` not working


### Additional Information

- Status version: develop 
- Operating System: Linux

It would be good to include some of the errors we got in the issue for future reference:
```
ℹ Building target typescript
ℹ Cleaning up previous build at lib/typescript
ℹ Generating type definitions with tsc
src/EmojiPicker.tsx:52:12 - error TS2322: Type '{ children: Element; style: { height: AnimatedSubtraction; }[]; }' is not assignable to type 'IntrinsicAttributes & { hitSlop?: WithAnimatedValue<Insets | undefined> | undefined; onLayout?: WithAnimatedValue<((event: LayoutChangeEvent) => void) | undefined> | undefined; ... 53 more ...; key?: WithAnimatedValue<...> | undefined; } & { ...; }'.
  Property 'children' does not exist on type 'IntrinsicAttributes & { hitSlop?: WithAnimatedValue<Insets | undefined> | undefined; onLayout?: WithAnimatedValue<((event: LayoutChangeEvent) => void) | undefined> | undefined; ... 53 more ...; key?: WithAnimatedValue<...> | undefined; } & { ...; }'.

52           <Animated.View
              ~~~~~~~~~~~~~

src/assets/Ban.tsx:5:4 - error TS2786: 'Svg' cannot be used as a JSX component.
  Its instance type 'Component<SvgProps, any, any>' is not a valid JSX element.

5   <Svg width="22" height="22" viewBox="0 0 22 22" fill="none">
     ~~~

src/assets/Ban.tsx:6:6 - error TS2786: 'Path' cannot be used as a JSX component.
  Its instance type 'Component<PathProps, any, any>' is not a valid JSX element.

6     <Path
       ~~~~
```
Full log: https://gist.github.com/jakubgs/7ffa6c414a0f65e98ef84c15c9f4d8c6

--- a/package.json
+++ b/package.json
@@ -71,7 +71,7 @@
     "react-native-svg": "^9.8.4",
     "react-native-touch-id": "^4.4.1",
     "react-native-webview": "git+https://github.com/status-im/react-native-webview.git#refs/tags/v11.16.0-status",
-    "rn-emoji-keyboard": "git+https://github.com/status-im/rn-emoji-keyboard.git#refs/tags/v0.4.2",
+    "rn-emoji-keyboard": "git+https://github.com/status-im/rn-emoji-keyboard.git#refs/tags/v0.4.3",
     "tdigest": "^0.1.1",
     "web3-utils": "^1.2.1"
   },

